modules/oadp-velero-timeouts.adoc

```diff
@@ -0,0 +1,33 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/application_backup_and_restore/troubleshooting.adoc
+
+:_content-type: PROCEDURE
+[id="Velero-timeout_{context}"]
+= Velereo resource timeout
+
+`resourceTimeout` defines how long to wait for several Velero resources before timeout occurs, such as Velero CRD availability, `volumeSnapshot` deletion, and repository availability. The default is `10m`.
```

Comment:
Consider spelling out CRD in this first instance (I see CR gets spelled out later):
```suggestion
`resourceTimeout` defines how long to wait for several Velero resources before timeout occurs, such as Velero custom resource definition (CRD) availability, `volumeSnapshot` deletion, and repository availability. The default is `10m`.
```